www/include/reporting/dashboard/xmlInformations/GetXmlService.php

```diff
@@ -43,7 +43,7 @@
 
 if (isset($_GET["host_id"]) && isset($_GET["id"]) && isset($_GET["color"])) {
```

Comment:
sanitize the arguments before the condition

Reply:
fix with https://github.com/centreon/centreon/pull/8419/commits/dd0c77b8ef23a0ab8f480a73e18ded83bf279095